core/src/main/scala/org/apache/spark/SparkContext.scala

```diff
@@ -816,6 +824,9 @@ class SparkContext(config: SparkConf) extends Logging {
   /**
    * Read a text file from HDFS, a local file system (available on all nodes), or any
    * Hadoop-supported file system URI, and return it as an RDD of Strings.
+   * @param path path to the text file on a supported file system
+   * @param minPartitions suggested minimum number of partitions for the resulting RDD
+   * @return RDD representing the content of the file
```

Comment:
I still might lean towards saying more, if saying anything at all here: it's an RDD of lines of the text file. Many of these comments are OK but minimal.